metric/system/process/process_test.go

```diff
@@ -118,6 +119,22 @@ func TestProcessList(t *testing.T) {
 	}
 }
 
+func TestSelfPersist(t *testing.T) {
+	stat, err := initTestResolver()
+	require.NoError(t, err, "Init()")
+	first, err := stat.GetSelf()
+	require.NoError(t, err, "First GetSelf()")
+
+	// The first process fetch shouldn't have percentages, since we don't have >1 procs to compare
+	assert.False(t, first.CPU.Total.Pct.Exists(), "total.pct should not exist")
+	time.Sleep(time.Millisecond * 5)
```

Comment:
Why 5 ms? Why is the delay here necessary?

Reply:
Interestingly enough, it depends on the OS. On darwin and windows, CI will fail because there's no time delta between the two measurements. The actual count was largely arbitrary.

Reply:
Just add a comment explaining what this is for then, the only other way I could see to have this work would be to mock time.Now() or something to control the time between calls.